bindata/v3.11.0/openshift-apiserver/ds.yaml

```diff
@@ -41,6 +41,8 @@ spec:
           name: client-ca
         - mountPath: /var/run/configmaps/etcd-serving-ca
           name: etcd-serving-ca
+        - mountPath: /var/run/configmaps/image-import-ca
+          name: image-import-ca
```

Comment:
should this be allowMissing ?

Reply:
?

I don't see that as a field on volumeMounts.  I don't think pods care if the configmap you're mounting doesn't exist, isn't that a key thing?

Reply:
ah, was needed on the volume section, not volumeMount.  added.